### ISSUE TITLE
[wavedeform] Tests don't play well with the 2016 GCD file (Trac #1897)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1889]
There are various failures. The old tests ran on a really old string 21 GCD file.  This is sub-optimal.  They should be able to run just fine on the latest 2016 GCD file.  I think handling of bad DOMs and new hardware accounts for a few of the failures, but not all.  I'll revisit this tomorrow, hopefully with Jim's help.

It seems like we should be able to make some fast solid tests too, that only calibrate one single DOM.

Migrated from https://code.icecube.wisc.edu/ticket/1897
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:24", 
    "description": "There are various failures. The old tests ran on a really old string 21 GCD file.  This is sub-optimal.  They should be able to run just fine on the latest 2016 GCD file.  I think handling of bad DOMs and new hardware accounts for a few of the failures, but not all.  I'll revisit this tomorrow, hopefully with Jim's help.\n\nIt seems like we should be able to make some fast solid tests too, that only calibrate one single DOM.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067204154158", 
    "component": "combo core", 
    "summary": "[wavedeform] Tests don't play well with the 2016 GCD file", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-10-13T22:07:57", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161103T17:31:28: 
* jbraun commented:

> Updated to only use the string 21 portion of the GCD.  This provides the same level of coverage as the old.  See #1671 for improving these tests.


* jbraun changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20161103T17:31:59: 
* jbraun changed resolution from "" to "fixed"
* jbraun changed status from "assigned" to "closed"
* jbraun commented:

> In changeset 151309:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="151309"
> Make tests compatible with the 2016 GCD file.  Updated to only use the string 21 portion of the GCD. This provides the same level of coverage as the old. I'll leave it to olivas to actually change the file.  Closes #1889.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:24: 
* olivas changed milestone from "October 2016" to ""
* olivas commented:

> Milestone October 2016 deleted
